lib/autoload/course/assessment/programming_test_case_report.rb

```diff
@@ -225,7 +225,11 @@ def messages
     # @return [String]
     def get_meta_attribute(attribute_name)
       meta = @test_case.search('meta')
-      meta.present? && meta[0][attribute_name] ? meta[0][attribute_name] : ''
+      if meta.present? && meta[0][attribute_name]
```

Comment:
Would it make more sense to have a `get_attribute` which then calls `get_meta_attribute`, and if that returns nil, we then use `@test_case[attribute_name] || ''`? 

Otherwise the `get_meta_attribute` function name here does not really reflect that it does - or we could just rename this to `get_attribute`